client/lib/domains/registration/availability-messages.js

```diff
@@ -258,4 +258,149 @@ function getAvailabilityNotice( domain, error, site ) {
 	};
 }
 
-export { getAvailabilityNotice };
+function getAvailabilityNoticeForTransfers(
+	domain,
+	status,
+	site,
+	rootDomain,
+	mappableStatus,
+	goToMapDomainStep
+) {
+	let message,
+		severity = 'error';
+
+	const isSubDomain = rootDomain && rootDomain !== domain;
+
+	switch ( status ) {
+		case domainAvailability.MAPPABLE:
+		case domainAvailability.TLD_NOT_SUPPORTED:
+			if ( isSubDomain ) {
+				message = translate(
+					'Transferring a subdomain is not possible. You can try to {{button}}map it{{/button}} instead ' +
+						'or transfer {{strong}}the root domain{{/strong}}.',
+					{
+						args: { rootDomain },
+						components: {
+							strong: <strong />,
+							button: <button onClick={ goToMapDomainStep } />,
+						},
+					}
+				);
+			} else {
+				const tld = getTld( domain );
+				message = translate(
+					"We don't support transfers for domains ending with {{strong}}.%(tld)s{{/strong}}, " +
+						'but you can {{button}}map it{{/button}} instead.',
+					{
+						args: { tld },
+						components: {
+							strong: <strong />,
+							button: <button onClick={ goToMapDomainStep } />,
+						},
+					}
+				);
+				severity = 'info';
+			}
+			break;
+		case domainAvailability.UNKNOWN:
+			if ( domainAvailability.MAPPABLE === mappableStatus ) {
+				message = translate(
+					"{{strong}}%(domain)s{{/strong}} can't be transferred. " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{strong}}%(domain)s{{/strong}} can't be transferred. You can {{a}}manually connect it{{/a}} if you still want to use it for your site.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).